pkg/apis/serving/v1alpha1/service_types.go

```diff
@@ -259,9 +259,15 @@ func (ss *ServiceStatus) PropagateConfigurationStatus(cs *ConfigurationStatus) {
 	}
 }
 
-// PropagateRouteStatus takes the Route status and applies its values
-// to the Service status.
-func (ss *ServiceStatus) PropagateRouteStatus(rs RouteStatus) {
+const (
+	trafficNotMigratedReason  = "Traffic not yet migrated"
```

Comment:
Reason should be a single camel-case work e.g. `TrafficNotMigrated`

Reply:
See [here](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#typical-status-properties)

Reply:
Done.